lib/skpm-build.js

```diff
@@ -158,7 +159,10 @@ function webpackConfig (file, commandIdentifier) {
         library: commandIdentifier ? 'onRun' : undefined,
         path: commandIdentifier
           ? path.join(output, 'Contents', 'Sketch')
-          : path.join(output, 'Contents', 'Resources')
+          : path.join(output, 'Contents', 'Resources'),
+        publicPath: `file://${commandIdentifier
+          ? path.join(output, 'Contents', 'Sketch')
```

Comment:
it's the same as above. Could you extract it in a variable and use it both in `path` and `publicPath`?